sda-commons-server-opentelemetry/src/main/java/org/sdase/commons/server/opentelemetry/autoconfig/SdaConfigPropertyProvider.java

```diff
@@ -20,6 +20,7 @@ private SdaConfigPropertyProvider() {
   public static Map<String, String> getProperties() {
     Map<String, String> properties = new HashMap<>();
 
+    properties.put("otel.traces.exporter", "jaeger");
```

Comment:
Do we have a test for expected default values?

Reply:
Now we have 😄 , thanks